src/runtime_src/core/pcie/driver/linux/xocl/subdev/qdma.c

```diff
@@ -1695,15 +1696,26 @@ static int qdma_probe(struct platform_device *pdev)
 	qdma->pdev = pdev;
 	platform_set_drvdata(pdev, qdma);
 
-	res = platform_get_resource(pdev, IORESOURCE_MEM, 0);
-	if (!res) {
-		xocl_err(&pdev->dev, "Empty resource");
-		return -EINVAL;
+	for (i = 0, res = platform_get_resource(pdev, IORESOURCE_MEM, 0);
+		res;	
+		res = platform_get_resource(pdev, IORESOURCE_MEM, ++i)) {
+		if (!strncmp(res->name, NODE_QDMA, strlen(NODE_QDMA))) {
```

Comment:
res->name is NULL for legacy 2RP device. Please add name to XOCL_RES_QDMA in devices.h

Reply:
will change